config_netlify.toml

```diff
@@ -0,0 +1,4 @@
+# Config settings for Netlify
+# https://discourse.gohugo.io/t/about-the-recent-redirect-loops-on-gohugo-io/13438
+
+disableAliases = true
```

Comment:
Since we are going to move over to using Netlify for deploying the production site should we just make this part of our core Hugo config in `config.toml`?

Reply:
I chatted with @IanMadd about this....until we recreate all of the aliases using an `_redirects` file we'll  want this config to only apply to the Netlify builds.